DataFormats/TrackerRecHit2D/interface/OmniClusterRef.h

```diff
@@ -39,9 +43,13 @@ class OmniClusterRef {
   }
 
   Phase2Cluster1DRef cluster_phase2OT()  const { 
-    return isPhase2() ? Phase2Cluster1DRef(me.toRefCore(),index()) : Phase2Cluster1DRef();
+    return (isPhase2() && !isTiming()) ? Phase2Cluster1DRef(me.toRefCore(),index()) : Phase2Cluster1DRef();
```

Comment:
Please keep it as it is.
```suggestion
    return isPhase2() ? Phase2Cluster1DRef(me.toRefCore(),index()) : Phase2Cluster1DRef();
```